i18n.d.ts

```diff
@@ -1,6 +1,8 @@
 import type en from "./messages/en.json";
+import type de from "./messages/de.json";
 
 type Messages = typeof en;
+type Messages = typeof de;
```

Comment:
@RedstonePfalz this is not needed, it's only about the shape

Reply:
I've removed the changes in i18n.d.ts.